openjdk8/pom.xml

```diff
@@ -77,7 +77,7 @@
 	      <goal>removeImage</goal>
 	    </goals>
 	    <configuration>
-	      <imageName>gke-debian-openjdk</imageName>
+	      <imageName>openjdk8</imageName>
```

Comment:
The name openjdk8 is a bit minimal for this artefact, which is a docker image customised for GKE with debian and openjdk8.      I find the name gke-debian-openjdk8 much more informative and not too long.